### PR TITLE
Implementing validation of number of objects in command 

### PR DESCRIPTION
**Pull request by dchen0258**
In relation with issue #357 . 
Goal: Implementing validation of number of objects in command. 
Changes: All changes made in src/operations.c For kind1 and 2 actions, checks to make sure there is not a third token in the array through token[2], which would indicate a presence of an extra object in the command.

### CODE REVIEW

File: src/cli/src/operations.c [hunk 1]
```diff
@@ -182,6 +186,10 @@ char *kind2_action_operation(char *tokens[TOKEN_LIST_SIZE], chiventure_ctx_t *ct
     {
         return "You must specify a direction to go \n";
     }
+    if(tokens[2] != NULL)
+    {
+        return "Sorry, You can only go one direction \n";
```
Comment (ecoble) [inline src/cli/src/operations.c:191]: Nit: "You" should be all lower case

**Review by ecoble — CHANGES_REQUESTED** [review]
Looks great! I have one very small change to request. Additionally, this branch will need to be updated since another team has merged into dev. You can do this within you project using git, but you can also do this on Github with the "Update branch" button at the bottom of the pull request screen.

File: src/cli/src/operations.c (hunk 1 above)
Comment (dchen0258) [reply src/cli/src/operations.c]: Problem should be fixed now!

**Review by ecoble — APPROVED** [review]
LGTM!

**Review by mjgoodman321 — APPROVED** [review]

**Comment by ecoble** [comment]
Issue Score: ✔️+

Comments:
Great job on this backlog issue! Just like issues, pull requests should also have closing statements. Also, be sure to keep the milestone up to date when it rolls over into the next sprint.

**Pull request merged**